from-dump/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quetzacoalt/docker-internal-images:1.7
+
+ENV PS_DOMAIN localhost
+ENV DUMP_PHP_FILES prestashop-dataset
+ENV DUMP_SQL_FILE prestashop-dataset.sql
+
+# Copy files
+COPY $DUMP_PHP_FILES/ /var/www/html/
+RUN ln -s /var/www/html/ /var/www/html/$DUMP_PHP_FILES/
```

Comment:
ping @aleeks for that point.

On PrestaShop, the domain & subfolder in which you installed PrestaShop is stored in the database. When you try to reach your shop with a non-matching URL and folder, it tries to redirect you to the proper address.

I can try to update the database in order to set the new URL during the build. We will see.